src/Link.js

```diff
@@ -27,8 +27,16 @@ class Link extends Component {
     this.onClick = this.onClick.bind(this);
   }
 
+  isAbsoluteURL(url) {
+    return url.startsWith("http://")
```

Comment:
I think a better way to check if a link is same domain or cross domain is:

```js
const link = this.refs.link;
location.hostname !== link.hostname;
```

because even if its an absolute url to the same domain we want to capture the click and use single-page-routing instead of a page reload.

Reply:
yes that definitely clarifies things.